app/mm-redux/actions/channels.ts

```diff
@@ -1475,6 +1475,26 @@ export function patchChannelModerations(channelId: string, patch: Array<ChannelM
     });
 }
 
+export function getChannelMemberCountsByGroup(channelId: string, includeTimezones: boolean): ActionFunc {
+    return async (dispatch: DispatchFunc) => {
+        let channelMemberCountsByGroup;
+        try {
+            channelMemberCountsByGroup = await Client4.getChannelMemberCountsByGroup(channelId, includeTimezones);
+        } catch (error) {
+            return {error};
+        }
+
+        if (channelMemberCountsByGroup) {
```

Comment:
can the client return `null` or `undefined` ? if not why the if statement ?

Reply:
No. Better do a `channelMemberCountsByGroup.length` here.